c/c-ares/stone.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# SPDX-FileCopyrightText: © 2020-2025 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : c-ares
+version     : v1.34.5
+release     : 1
+homepage    : https://c-ares.org/
+upstreams   :
+    - https://github.com/c-ares/c-ares/releases/download/v1.34.5/c-ares-1.34.5.tar.gz : 7d935790e9af081c25c495fd13c2cfcda4792983418e96358ef6e7320ee06346
+summary     : C library that performs DNS requests and name resolves asynchronously
+description : |
+    This is c-ares, an asynchronous resolver library. It is intended for applications which need to perform DNS queries without blocking, or need to perform multiple DNS queries in parallel. The primary examples of such applications are servers which communicate with multiple clients and programs with graphical user interfaces.
+license     : MIT
+builddeps   :
+    - pkgconfig(gtest)
+setup       : |
+    %patch %(pkgdir)/Remove-live-tests.patch
+    %configure
```

Comment:
Probably needs `--disable-static` to get rid of that `.a` static lib.

Reply:
Actually, it looks like this can be built with cmake. Is there a particular reason for using the autotools build here?